elements/pf-code-block/test/pf-code-block.spec.ts

```diff
@@ -13,12 +13,12 @@ const element = html`
       connectionConfig:
       url: https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs
     </script>
-    <pfe-clipboard slot="actions" copy-from="#code"></pfe-clipboard>
-  </pfe-code-block>
+    <pf-clipboard slot="actions" copy-from="#code"></pf-clipboard>
```

Comment:
`<pf-clipboard ...></pf-clipboard>` should be `<pf-clipboard-copy ...></pf-clipboard-copy>`